recipes/static/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "static" %}
+{% set version = "1.1.1" %}
+{% set build = 2 %}
```

Comment:
Unless you are trying to avoid a collision with a version present in the default channel let's set this one to `0`.